python/lsst/sims/catUtils/galSimInterface/galSimCatalogs.py

```diff
@@ -81,9 +72,21 @@ class similar to drawPoinSource and drawSersic)
     for bpn in bandpass_names:
         name = self.bandpass_directory+'/'+self.bandpass_root+'_'+bpn+'.dat'
 
-    4) Telescope parameters such as exposure time, area, and gain are controled by the
-    GalSim InstanceCatalog member variables exposure_time (in s), effective_area (in cm^2),
-    and gain (in photons per ADU)
+    one should also define the following member variables:
+
+        componentList is a list of files ins banpass_directory containing the response
+        curves for the different components of the camera, e.g.
+        ['detector.dat', 'm1.dat', 'm2.dat', 'm3.dat', 'lens1.dat', 'lens2.dat', 'lens3.dat']
+
+        skyBandpassName is the name of the file in bandpass_directory that contains the sky
```

Comment:
Niggly comment about this docstring. I think if you call it 'atmospheric transmission curve' that it will be clearer what you actually want here (sky response curve doesn't actually mean anything obvious to me). So maybe update the rest of the string and change skyBandpassName to atmoBandpassName?